src/sentry/incidents/subscription_processor.py

```diff
@@ -358,6 +361,25 @@ def process_update(self, subscription_update: QuerySubscriptionUpdate) -> None:
             metrics.incr("incidents.alert_rules.skipping_already_processed_update")
             return
 
+        if features.has(
+            "organizations:workflow-engine-m3-dual-write", self.subscription.project.organization
+        ):
+            # NOTE: feed the data through the new pipeline, but don't do anything with it yet.
+            # This will change at some point.
+            data_packet = DataPacket(
+                query_id=self.subscription.snuba_query.id, packet=subscription_update
+            )
+            detectors = process_data_sources([data_packet], query_type=self.subscription.type)
```

Comment:
🤔 - i was intending the `query_type` to be more of the `DataSource`, so this would be like `snuba_subscription` or something along those lines. maybe we should have those exported from `data_sources` as an enum and applied to the `type` field.

Reply:
I think Colleen is writing an enum for this purpose!

Reply:
TODO: change `query_type` to be `DataSourceType.SNUBA_QUERY_SUBSCRIPTION` once Colleen's PR lands